src/utils/listener/listener.go

```diff
@@ -196,6 +198,15 @@ func (self *Listener) monitorBlocks() {
 				// FIXME: Inform downstream something's wrong
 			}
 
+			if !block.IsValid() {
```

Comment:
how exactly will it currently behave? retry-ing to validate the same block from the same peer doesn't make much sense..?

Reply:
This will be handled in this issue: https://github.com/warp-contracts/syncer/issues/9